contracts/XVSVault/XVSVault.sol

```diff
@@ -845,6 +864,25 @@ contract XVSVault is XVSVaultStorage, ECDSA, AccessControlledV5 {
         emit StoreUpdated(oldXvsContract, oldStore, _xvs, _xvsStore);
     }
 
+    /**
+     * @notice Sets the address of the prime token contract
+     * @param _primeToken address of the prime token contract
+     * @param _primeRewardToken address of reward token
+     * @param _primePoolId pool id for reward
+     */
+    function setPrimeToken(IPrime _primeToken, address _primeRewardToken, uint256 _primePoolId) external {
```

Comment:
I think this should be restricted to `onlyOwner`, not using the ACM for this. Rationale: it's setting a contract relationship; it's should not be needed to change this using a Fast-track or Critical VIP, and we would reduce the risk not using the ACM